src/containers/NahmiiHoc/saga.js

```diff
@@ -188,7 +188,7 @@ export function* getSdkWalletSigner(wallet) {
     const deviceId = trezorInfo.get('id');
     const path = wallet.derivationPath;
     const publicAddressKeyPair = yield call(requestHardwareWalletAPI, 'getaddress', { id: deviceId, path });
-    if (!isAddressMatch(`0x${publicAddressKeyPair.address}`, wallet.address)) {
+    if (!isAddressMatch(prependHexToAddress(publicAddressKeyPair.address), wallet.address)) {
```

Comment:
Could use "prefix0x()" from the SDK

Reply:
Thanks for the tips. I will use the prefix0x from SDK in next PR.